website/docs/developer-docs/docs/style-guide.mdx

```diff
@@ -270,60 +311,93 @@ import TabItem from "@theme/TabItem";
+- **Possible causes**:
+    - Incorrect username or password.
+    - Account locked due to multiple failed attempts.
 
-- **Solution:**
+- **Solutions**:
+    1. Verify your credentials.
+    2. Reset your password using the **Forgot Password** link.
+    3. Contact your administrator if the account is locked.
 
-    - Verify your username and password
-    - Reset your password if necessary
-    - Contact an administrator if your account is locked
+---
 
 ## Accessibility best practices
 
-- Avoid using color as the sole method of conveying information (e.g., "Click the red button"). Instead, use descriptive labels to ensure accessibility.
-- Provide **descriptive link text**. Avoid using generic terms like "Click here". Be specific about where the link will take the user.
-    - **DON'T:** "Click here."
-    - **DO:** "See the [Authentication Settings](/) for more details."
+- **Alt text for images**: Describe the purpose of the image, not just its appearance. For example, use "Screenshot of the login form" instead of "Image of a form."
+- **Heading hierarchy**: Use headings in order (H1 → H2 → H3) to support screen readers.
 
-## Notes and warnings
+---
 
-For notes and important information:
+## Notes and warnings
 
-**Notes** are formatted using:
+Use the following components to highlight important information:
 
-```
+**Info notes**:
+```md
 :::info
-Write your note here.
+This is a tip or general note.
 :::
 ```
 
-**Warnings** are formatted using:
-
-```
+**Warnings**:
+```md
 :::warning
-Write your warning here.
+This level is for more serious situations: an action cannot be undone, a process might be canceled, etc..
+:::
+```
+
+**Critical warnings** (for irreversible actions):
+```md
+:::danger
+This level is for extremely serious situations, such as an action permanently removing data.
 :::
 ```
 
-Use these conventions to ensure that the reader's attention is drawn to crucial information.
+---
+
+## Inclusive Language
+
+- Use **gender-neutral pronouns** like "they/them" instead of "he/she" (e.g., "The user should check their settings").
+- Avoid **ableist terms** such as "dumb" or "lame"; use "non-functional" or "unavailable" instead.
+- **Avoid idioms** that may not translate well (e.g., instead of "hit a home run" use "achieve success").
+
+---
+
+## Images and Media
+
+- **Screenshots**:
+    - Update screenshots with UI changes.
+    - Crop to focus on relevant elements.
+    - Add descriptive alt text (e.g., "Screenshot of the Provider configuration page").
+- **Diagrams**: Use tools like [Draw.io](https://draw.io) for clarity. Ensure high contrast and text descriptions. todo(dominic): mention mermaid, might be too complicated for some? if even possible here
+
+---
+
+## Document Structure and Metadata
+
+- **Front matter**: Include a title and optional summary:
+    ```md
+    ---
+    title: Getting Started
+    description: Install and configure authentik in 5 minutes.
```

Comment:
Welp, you really opened a can of worms here. :-) You can also explain how to add badges... like for version number or if it is Enterprise. For Integration guides, we have the badge showing level of support.